src/main/scala/nl.knaw.dans.easy.bag2deposit/DepositPropertiesFactory.scala

```diff
@@ -39,6 +40,21 @@ case class DepositPropertiesFactory(configuration: Configuration, idType: IdType
       .getOrElse(throw InvalidBagException("no fedoraID"))
       .text
 
+    def nbn = bagInfo.versionOf.map(
+      configuration.bagIndex.getURN(_).unsafeGetOrThrow
+    ).getOrElse(urn)
+
+    def checkSequence(): Unit = {
+      val seqLength = configuration.bagIndex
+        .getSeqLength(bagInfo.uuid)
+        .getOrRecover {
+          case InvalidBagException(_) => 0 // not found
```

Comment:
@lindareijnhoudt 
Or do you want it to be a failure when a bag is not found in the index?

By the way, when a bag throws an error, the whole batch is terminated: https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/f619163232854083108a02f51ecf13b11c00ed37/src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDespositApp.scala#L35-L37